tests/http_bench.ts

```diff
@@ -5,14 +5,22 @@ import * as deno from "deno";
 const addr = deno.args[1] || "127.0.0.1:4500";
 const listener = deno.listen("tcp", addr);
 const response = new TextEncoder().encode(
-  "HTTP/1.1 200 OK\r\nContent-Length: 12\r\n\r\nHello World\n"
+  "HTTP/1.1 200 OK\r\nContent-Length: 12\r\nConnection: keep-alive\r\n\r\nHello World\n"
```

Comment:
keep-alive is default in http 1.1

Reply:
removed - indeed, it doesn't seem to effect the bench.